src/components/SignUp/Landing.tsx

```diff
@@ -202,12 +237,74 @@ const Landing: React.FC<{
 };
 
 const StyledCard = styled(Col)`
+  width: 273px;
+  --cardHeight: 353.23px;
+  height: var(--cardHeight);
   margin-right: 20px;
   box-shadow: 2px 2px 10px rgba(0, 0, 29, 0.1);
   border-radius: 8px;
   background-color: ${colors.bodyBg};
+  backdrop-filter: blur(15px);
   padding: 30px 20px 50px 20px;
-  height: 100%;
+
+  div {
+    overflow: auto;
+    height: calc(var(--cardHeight) / 2);
+    margin-top: 5px;
+
+    div:last-of-type {
+      position: absolute;
+      display: flex;
+      align-items: self-start;
+    }
+  }
+
+  p {
+    left: 0%;
+    right: 4.5%;
+    top: 45%;
+    bottom: 0%;
+
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 24px;
+    /* or 150% */
+    text-align: left;
+    letter-spacing: 0.5px;
+
+    color: #000000;
+
+    /* Inside Auto Layout */
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+    margin: 12px 0px;
+  }
+
+  .cardHeading {
+    position: static;
```

Comment:
Another example of properties we can leave out if they're default. Sometimes Figma will add _all_ of the properties and we only need a few of them. Always good to start small with the bare minimum and add properties if necessary (especially when it comes to positioning of the elements)